client/keystore/src/local.rs

```diff
@@ -142,9 +156,18 @@ impl Keystore for LocalKeystore {
 		&self,
 		key_type: KeyTypeId,
 		public: &sr25519::Public,
-		transcript: &sr25519::vrf::VrfTranscript,
+		input: &sr25519::vrf::VrfInput,
 	) -> std::result::Result<Option<sr25519::vrf::VrfSignature>, TraitError> {
-		self.vrf_sign::<sr25519::Pair>(key_type, public, transcript)
+		self.vrf_sign::<sr25519::Pair>(key_type, public, input)
+	}
+
+	fn sr25519_vrf_preout(
+		&self,
+		key_type: KeyTypeId,
+		public: &sr25519::Public,
+		input: &sr25519::vrf::VrfInput,
+	) -> std::result::Result<Option<sr25519::vrf::VrfPreOutput>, TraitError> {
+		self.vrf_preout::<sr25519::Pair>(key_type, public, input)
```

Comment:
I cannot figure out how this method is useful in the interface provided.  It's not so much that this method does not work, as alone it does not enable anything new.  We can chat voice or on element about it I guess.

Reply:
@burdges sure, we can have a chat about it tomorrow. I'll contact you on element